app/services/api/conversions/create_project_service.rb

```diff
@@ -1,26 +1,51 @@
 class Api::Conversions::CreateProjectService
+  include ActiveModel::Model
+  include ActiveModel::Attributes
+  include ActiveRecord::AttributeAssignment
+
   class ProjectCreationError < StandardError; end
 
+  attribute :urn
+  attribute :incoming_trust_ukprn
+  attribute :advisory_board_date
+  attribute :advisory_board_conditions
+  attribute :provisional_conversion_date
+  attribute :directive_academy_order
+  attribute :created_by_email
+  attribute :created_by_first_name
+  attribute :created_by_last_name
+
+  validates :urn, presence: true, urn: true
+  validate :establishment_exists
```

Comment:
There is a decision here for me about how far we trust the data being posted from Prepare, in theory the establishment and trust MUST exist so do we need this? What is our stance, should we document it?

Reply:
This is fair, happy to get rid of it tbh!!